### ISSUE TITLE
Enter text in blank input not allowed in Internet Explorer (Mix-mode)

### ISSUE THREAD

**Issue by singh-vikramjit** [yairEO/tagify#440]
## Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Demo Page

https://jsbin.com/nagocireha/edit?html,css,js,output

### Explanation
Yesterday I test my code in Internet Explorer 11 and found a few things which are not working as expected.

1. When I click inside the blank input to add a text and tag it does not enable the cursor to write something. ( face problem in Internet Explorer 11, Edge (Microsoft Edge 44.17763.831.0) )

2. And when I click inside the input which has predefined value then it works fine but sometimes when I click it takes the cursor to the starting point of the input text instead of at the end of the text.
(for this it would be great if check the attached file)
[sample-vue.zip](https://github.com/yairEO/tagify/files/4297974/sample-vue.zip)

3. And some time tag in mix mode does not work properly like I am using # to make a new tag and when I press # and the start type something then nothing happens and when I press backspace once then it indicates the tags dropdown (you can check in code snippet and in the below screenshot for better insight)
![ezgif com-video-to-gif (2)](https://user-images.githubusercontent.com/45350698/76078093-42e70b00-5fa2-11ea-9e46-0a9b26afddcd.gif)

4. when I perform focus event the cursor goes to the starting point instead of the end of the text. when you click on the buttons (tag and ux-team) then you can check cursor position. ( face problem in Internet Explorer 11, Edge (Microsoft Edge 44.17763.831.0) )

Please check these issues and share your thoughts ASAP.

**Comment by singh-vikramjit** [comment]
Hi @yairEO have you checked these issues?

**Comment by yairEO** [comment]
Yes, I am trying to understand why and how to fix this.

**Comment by singh-vikramjit** [comment]
Hi @yairEO Thanks for these fixes but I tried with new CSS and js file and 3rd issue is still there.

3. And some time tag in mix mode does not work properly like I am using # to make a new tag and when I press # and the start type something then nothing happens and when I press backspace once then it indicates the tags dropdown (you can check in the code snippet and in the below screenshot for better insight)

can you please check this issue once again. because now it is more frequent now every time when I press # and start typing nothing will happen every time when I press backspace then it works.

**Comment by singh-vikramjit** [comment]
I found one more issue with the latest version in **Firefox** 

when I click on input witch has text and ending with a Tag not able to focus and type something

https://jsbin.com/nagocireha/edit?html,css,js,output

if you click in the middle of the input where the normal text is written then cursor display and when click at the end to type something cursor is not displayed and not able to write something

**Comment by yairEO** [comment]
Yes I accidently closed this issue, it will be fixed shortly, trust me :)

**Comment by singh-vikramjit** [comment]
regarding the above **Firefox** issue I found an interesting thing like when i click at the end cursor is not displayed and i am not able to write something but if i press backspace the tag is removed and the cursor is visible and then create a new tag then focus out and again focus in input and then it works 

the issue is only with the predefined default value 

![ezgif com-video-to-gif (4)](https://user-images.githubusercontent.com/45350698/76948825-e839a180-6907-11ea-8947-b5a1e59dfbe9.gif)

**Comment by yairEO** [comment]
Thanks for your excellent reports I keep finding more bugs. I am on it. As these days with the Corona virus around, I somehow have less time to work, but still do, don't worry

**Comment by singh-vikramjit** [comment]
Hi, @yairEO thanks for the new version. but I test with the latest version cart at the end fix in firefox but still remain in Edge in a different way. like when you focus on the element which is ending with a tag then carat is displayed at the starting point of the tag and you can not move it to end of the tag with arrow keys as well so if you want to write something more then you need to press the delete button to remove that tag then you are able to write something

**Comment by yairEO** [comment]
Thanks for letting me know, I forgot to test in Edge, I will work on a fix today

**Comment by singh-vikramjit** [comment]
and one small issue in IE-11 like when I try to select a tag from the whitelist dropdown I am not able to select the tag because when I press down arrow not able to select tag you can check in this jsbin

https://jsbin.com/zarahoxemo/1/edit?html,js,output

**Comment by yairEO** [comment]
I see.. This only seems to happen if there's no text after the last tag.

**Comment by yairEO** [comment]
MS Edge bug is very weird and I cannot find any Microsoft online bug tracker to see if there's a reference to it. I've tried Google without luck.

I've made a minimal [demo to showcase the bug](https://jsbin.com/maluxog/edit?html,css,output)
